dark/civ/proteins.py

```diff
@@ -1866,6 +1900,22 @@ def _addGenomes(
             else:
                 taxonomyId = lineage[0][0]
 
+            if allowedTaxonomicRanks:
+                if lineage:
+                    if not isAllowedTaxonomicRank(allowedTaxonomicRanks, lineage):
+                        if logfp:
+                            print(
+                                "  %s (%s) has an unwanted taxonomic lineage."
+                                % (genome.id, genome.description),
+                                file=logfp,
+                            )
+                        continue
+                else:
+                    # We are allowing only certain taxonomic ranks and we
```

Comment:
Why is this not logged?

Also sometimes in this function you log "skipping..." but then don't follow with a `continue`

Reply:
Thanks! I added a logging print, and have made all the `continue`s in that function say "skipping" and removed the "skipping" that didn't have a `continue`.